Deployment

elbotho opened the pull request:
<!-- changes-start -->
---
## Changes:

### apps
- #4531 @elbotho
- #4528 @LarsTheGlidingSquirrel
- #4520 @dependabot
- #4525 @elbotho
- #4521 @elbotho
### packages
- f47b8744fa9484b3de4c7a1630db3c3e986b5e1b
- 66b5ead33df4bb77146c5204330154c9a9ba5d7c
- #4528 @LarsTheGlidingSquirrel
- #4526 @hejtful
- #4524 @elbotho
- dcb77bb65167744f12321006e34bcd1e8f2486d4
- #4363 @hejtful
### e2e-tests
### Others
- #4530 @dependabot
- #4529 @dependabot
- #4524 @elbotho
<!-- changes-end -->

hejtful committed:
feat(editor-ui): onboarding welcome modal and hook

hejtful committed:
refactor(editor-ui): simplify welcome modal

hejtful committed:
feat(editor-ui): add welcome modal video and text content

hejtful committed:
fix(editor-ui): welcome modal align padding

hejtful committed:
fix(editor-ui): welcome modal text changes

hejtful committed:
fix(editor-ui): welcome modal localStorage key naming convention

elbotho committed:
fix(editor-ui): improve text

elbotho committed:
fix: move exclamation mark

dependabot[bot] committed:
chore(deps): bump mathlive from 0.101.1 to 0.104.0

Bumps [mathlive](https://github.com/arnog/mathlive) from 0.101.1 to 0.104.0.
- [Release notes](https://github.com/arnog/mathlive/releases)
- [Changelog](https://github.com/arnog/mathlive/blob/master/CHANGELOG.md)
- [Commits](https://github.com/arnog/mathlive/commits)

---
updated-dependencies:
- dependency-name: mathlive
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

elbotho committed:
feat(web): use language in preview

elbotho committed:
feat(web): add editor-only mode (no preview)

elbotho committed:
refactor(web): change header nav position and style

elbotho committed:
refactor(web): showUndoRedoButtons in preview

elbotho committed:
feat(web): add preview only mode as well

elbotho committed:
chore(web): limit maxwidth of preview

elbotho committed:
Merge pull request #4522 from serlo/production

Merge pull request #4515 from serlo/staging

elbotho committed:
Merge pull request #4521 from serlo/feat/new-preview-features

feat(web): new features for editor preview

elbotho committed:
refactor(editor): add i18n to upload/validation errors

elbotho committed:
refactor(editor): split validation and validation error handling

elbotho committed:
refactor(web): some styled jsx to tailwind

elbotho committed:
Merge pull request #4525 from serlo/refactor/remove-some-style-jsx

refactor(web): some styled jsx to tailwind

elbotho committed:
Merge pull request #4524 from serlo/refactor/upload-error-validate

refactor(editor): i18n for validation errors and some refactoring

elbotho committed:
Merge pull request #4520 from serlo/dependabot/npm_and_yarn/mathlive-0.104.0

chore(deps): bump mathlive from 0.101.1 to 0.104.0

hejtful committed:
chore: bump editor and editor-web-component versions

hejtful committed:
Merge pull request #4526 from serlo/chore/editor-v0.23.0

chore: bump editor and editor-web-component versions

elbotho committed:
refactor(web): update extraMetaTags

LarsTheGlidingSquirrel committed:
revert a5956363cf5e9334f08fd4f7d274bf8569f1bade

LarsTheGlidingSquirrel committed:
Merge pull request #4528 from serlo/revert-iframe-resizer-changes

Revert contentOnly pages: Use old iframe resizing method using `iframe-resizer` instead of `open-iframe-resizer`

dependabot[bot] committed:
chore(deps): bump @babel/helpers from 7.23.6 to 7.26.10

Bumps [@babel/helpers](https://github.com/babel/babel/tree/HEAD/packages/babel-helpers) from 7.23.6 to 7.26.10.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.10/packages/babel-helpers)

---
updated-dependencies:
- dependency-name: "@babel/helpers"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

LarsTheGlidingSquirrel commented:
Can we do the deployment today / tomorrow? Would be great to have the RLP bugfix deployed by Tuesday evening. :relaxed:

dependabot[bot] committed:
chore(deps): bump @babel/runtime from 7.23.5 to 7.26.10

Bumps [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) from 7.23.5 to 7.26.10.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.10/packages/babel-runtime)

---
updated-dependencies:
- dependency-name: "@babel/runtime"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

elbotho committed:
Merge pull request #4531 from serlo/meta-tag-updates

refactor(web): update extraMetaTags

elbotho committed:
Merge pull request #4529 from serlo/dependabot/npm_and_yarn/babel/helpers-7.26.10

chore(deps): bump @babel/helpers from 7.23.6 to 7.26.10

elbotho committed:
Merge pull request #4530 from serlo/dependabot/npm_and_yarn/babel/runtime-7.26.10

chore(deps): bump @babel/runtime from 7.23.5 to 7.26.10

hejtful committed:
Merge branch 'staging' into feat/onboarding-welcome-modal

hejtful committed:
chore(welcome-modal): temp hide second step

hejtful committed:
Merge pull request #4363 from serlo/feat/onboarding-welcome-modal

feat(editor-ui): onboarding welcome modal and hook
